ublox/src/ubx_packets/packets.rs

```diff
@@ -1903,8 +1907,6 @@ struct MonVer {
 }
 
 mod mon_ver {
-    use super::MonVerExtensionIter;
-
     pub(crate) fn convert_to_str_unchecked(bytes: &[u8]) -> &str {
```

Comment:
should we refactor this `mod` else where like how we have done with putting things in `impl ...`
would probs need to move the test as well (wasn't sure if there was a purpose to putting this in mod and pub(crate) fn)